### PR TITLE
DOC: update Readme

### PR DESCRIPTION
**Pull request by tupui**
Update the Readme to reflect NumPy's readme:

* SciPy logo
* NumFOCUS logo
* A more detailed contributing section (same as NumPy)

And extra things:
 
* A link to the articles for citation
* DOI badge with link to SciPy's article
* A StackOverflow badge to ask questions
* Remove travis badge

(See the proposed changes on my fork)

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -1,9 +1,9 @@
+.. image:: doc/source/_static/scipyshiny_small.png
+  :target: https://docs.scipy.org/doc/scipy/reference/
```
Comment (rgommers) [inline README.rst:2]: This is very large, could you make it a couple of times smaller?

File: README.rst [hunk 2]
```diff
@@ -47,9 +55,41 @@ SciPy a try!
...
+
+Call for Contributions
+----------------------
+
+The SciPy project welcomes your expertise and enthusiasm!
+
+Small improvements or fixes are always appreciated; issues labeled as "good
+first issue" may be a good starting point. If you are considering larger
+contributions to the source code, please contact us through the [mailing
+list](https://scipy.org/scipylib/mailing-lists.html) first.
```
Comment (rgommers) [inline README.rst:67]: These are Markdown-style links in a `.rst` file, they don't work (a few more further down).

File: README.rst [hunk 3]
```diff
@@ -47,9 +55,41 @@ SciPy a try!
...
-SciPy development, take a look at the file CONTRIBUTING.rst_.
+
+Call for Contributions
+----------------------
+
+The SciPy project welcomes your expertise and enthusiasm!
+
+Small improvements or fixes are always appreciated; issues labeled as "good
+first issue" may be a good starting point. If you are considering larger
+contributions to the source code, please contact us through the [mailing
```
Comment (rgommers) [inline README.rst:66]: This sentence I don't quite like anymore. Usually GitHub is the right avenue, and larger bug fixes don't need discussion. Can you just link to http://scipy.github.io/devdocs/dev/hacking.html instead?

File: README.rst [hunk 4]
```diff
@@ -47,9 +55,41 @@ SciPy a try!
...
+list](https://scipy.org/scipylib/mailing-lists.html) first.
+Have a look at our contributing guide CONTRIBUTING.rst_.
```
Comment (rgommers) [inline README.rst:68]: This link should be removed.

File: README.rst [hunk 5]
```diff
@@ -47,9 +55,41 @@ SciPy a try!
...
+
+Our preferred channels of communication are all public.
+
+If you are new to contributing to open source, [this
+guide](https://opensource.guide/how-to-contribute/) helps explain why, what,
+and how to successfully get involved.
 
 
 .. _CONTRIBUTING.rst:  https://github.com/scipy/scipy/blob/master/CONTRIBUTING.rst
 .. _INSTALL.rst.txt:   https://github.com/scipy/scipy/blob/master/INSTALL.rst.txt
```
Comment (rgommers) [inline README.rst:92]: Better to link to the html docs for these too.

**Review by rgommers — CHANGES_REQUESTED** [review]
Thanks @tupui, good idea to update the README. Overall looks good, some things to change.

File: README.rst (hunk 2 above)
Comment (tupui) [reply README.rst:67]: 🤦  indeed! I will fix these.

File: README.rst (hunk 1 above)
Comment (tupui) [reply README.rst:2]: I can and I will try to use also `:align: left`

Comment (tupui) [reply README.rst:2]: Apparently I need raw html here.

**Comment by tupui** [comment]
I updated the links. Let me know if this is correct. I am not sure about which link to use for the install instructions as we have quite some places.

File: README.rst (hunk 1 above)
Comment (tupui) [reply README.rst]: It works 😃

**Comment by tupui** [comment]
Sorry for the force pushes. I tried to align the logo but seemingly it does not change anything. So I removed the style in the end.

File: README.rst [hunk 6]
```diff
@@ -45,11 +56,39 @@ upon by some of the world's leading scientists and engineers. If you need to
...
 
-For the installation instructions, see INSTALL.rst.txt_.
+For the installation instructions, see `our install
+guide <https://scipy.github.io/devdocs/getting_started.html#installation>`__.
+
+
+Call for Contributions
+----------------------
+
+The SciPy project welcomes your expertise and enthusiasm!
```
Comment (ilayn) [inline README.rst:66]: We are not calling like a conference anyways so this is a bit academic. People who have none of these are also welcome. Probably we need a more neutral language here. I think the old sentence can be used here.

File: README.rst [hunk 7]
```diff
@@ -22,18 +25,26 @@ SciPy
...
 - **Documentation:** https://docs.scipy.org/
 - **Mailing list:** https://scipy.org/scipylib/mailing-lists.html
 - **Source code:** https://github.com/scipy/scipy
+- **Contributing:** https://scipy.github.io/devdocs/dev/index.html
 - **Bug reports:** https://github.com/scipy/scipy/issues
 - **Code of Conduct:** https://scipy.github.io/devdocs/dev/conduct/code_of_conduct.html
 - **Report a security vulnerability:** https://tidelift.com/docs/security
+- **Citing in your work:** https://www.scipy.org/citing.html
 
 SciPy depends on NumPy, which provides convenient and fast
```
Comment (ilayn) [inline README.rst:49]: NumPy detail is not needed here anymore. Since the next one already mentions it

File: README.rst [hunk 8]
```diff
@@ -45,11 +56,39 @@ upon by some of the world's leading scientists and engineers. If you need to
...
+- review pull requests
+- triage issues
+- develop tutorials, presentations, and other educational materials
+- maintain and improve `our website <https://github.com/scipy/scipy.org>`__
+- develop graphic design for our brand assets and promotional materials
+- translate website content
+- help with outreach and onboard new contributors
+- write grant proposals and help with other fundraising efforts
+
+If you’re unsure where to start or how your skills fit in, reach out! You can
```
Comment (ilayn) [inline README.rst:83]: We don't want meta issues in the issues. Probably we can use the "Discussions" instead or the mailing list.

File: README.rst [hunk 9]
```diff
@@ -45,11 +56,39 @@ upon by some of the world's leading scientists and engineers. If you need to
...
+ask on the mailing list or here, on GitHub, by opening a new issue or leaving a
+comment on a relevant issue that is already open.
 
-We appreciate and welcome contributions. If you would like to take part in
-SciPy development, take a look at the file CONTRIBUTING.rst_.
+Our preferred channels of communication are all public.
```
Comment (ilayn) [inline README.rst:87]: What do you mean by this?

File: README.rst [hunk 10]
```diff
@@ -45,11 +56,39 @@ upon by some of the world's leading scientists and engineers. If you need to
...
 
+If you are new to contributing to open source, `this
+guide <https://opensource.guide/how-to-contribute/>`__ helps explain why, what,
+and how to successfully get involved.
```
Comment (ilayn) [inline README.rst:91]: successfully?

File: README.rst (hunk 6 above)
Comment (tupui) [reply README.rst:66]: Thanks for having a look! I took all of this from NumPy, so sure we can adapt it as we want. I just like to be consistant between the 2. What would you propose here?

File: README.rst (hunk 8 above)
Comment (tupui) [reply README.rst:83]: Then I propose to remove `opening a new issue or `. Then if says, either use the mailing list or comment on issues.

File: README.rst (hunk 9 above)
Comment (tupui) [reply README.rst:87]: That there is no back-channel and everything we do is public and you can audit it. But again this is from NumPy.

File: README.rst (hunk 10 above)
Comment (tupui) [reply README.rst:91]: NumPy again, but I like this kind of motivational speech.

**Comment by tupui** [comment]
Thanks @ilayn, I integrated some changes to answer your review. Let me know if this is better and for the rest.

File: README.rst (hunk 10 above)
Comment (ilayn) [reply README.rst:90]: Implies there is an option of failure though there is no such metric involved. The newcomers are often very scared so let's not tap into that fear for no reason.

File: README.rst (hunk 6 above)
Comment (ilayn) [reply README.rst:65]: The old sentence (removed now) is pretty OKish and neutral don't you think?

File: README.rst (hunk 9 above)
Comment (ilayn) [reply README.rst:86]: Exactly what a back-channel owner would say 😏

File: README.rst [hunk 11]
```diff
@@ -45,11 +55,39 @@ upon by some of the world's leading scientists and engineers. If you need to
...
+guide <http://scipy.github.io/devdocs/dev/hacking.html>`__.
+
+Writing code isn’t the only way to contribute to SciPy. You can also:
+
+- review pull requests
+- triage issues
+- develop tutorials, presentations, and other educational materials
+- maintain and improve `our website <https://github.com/scipy/scipy.org>`__
+- develop graphic design for our brand assets and promotional materials
+- translate website content
```
Comment (tylerjereddy) [inline README.rst:78]: Not trying to discourage it, but NumPy seems to have a bit of discussion about the amount of work it is to maintain translations: https://numpy.org/neps/nep-0028-website-redesign.html#translation-multilingual-i18n

I wonder if they ended up sticking with one of the rules listed there: 

> The language must have a dedicated maintainer

Ralf would probably know--I've seen them on a bunch of the language PRs.

Comment (rgommers) [reply README.rst:78]: Oh yes, this should be removed here. We will translate the NumPy website, but not the docs. And for SciPy we first need a new website ....

Comment (tupui) [reply README.rst:78]: Ok sure I will remove it, thanks

File: README.rst (hunk 9 above)
Comment (tupui) [reply README.rst]: I removed but for reference this is the inspiration

> Our preferred channels of communication are all public, but if you’d like to speak to us in private first, contact our community coordinators at numpy-team@googlegroups.com or on Slack (write numpy-team@googlegroups.com for an invitation).
>We also have a biweekly community call, details of which are announced on the mailing list. You are very welcome to join.

And we probably should have something like this. But that's another discussion.

File: README.rst (hunk 10 above)
Comment (tupui) [reply README.rst]: I removed the work `successfully`

**Comment by tupui** [comment]
Does anyone have other suggestions?

**Review by ilayn — APPROVED** [review]

**Review by rgommers — APPROVED** [review]
LGTM too now, thanks @tupui!

**Pull request merged**